frontend/src/pages/WorkspaceLocked/workspaceLocked.data.ts

```diff
@@ -0,0 +1,156 @@
+export const infoData = [
+	{
+		id: 'infoBlock-1',
+		title: 'Built for scale',
+		description:
+			'Our powerful ingestion engine has a proven track record of handling 10TB+ data ingestion per day.',
+	},
+	{
+		id: 'infoBlock-2',
+		title: 'Trusted across the globe',
+		description:
+			'Used by teams in all 5 continents ⎯ across the mountains, rivers, and the high seas.',
+	},
+	{
+		id: 'infoBlock-3',
+		title: 'Powering observability for teams of all sizes',
+		description:
+			'Hundreds of companies ⎯from early-stage start-ups to public enterprises use SigNoz to build more reliable products.',
+	},
+];
+
+export const enterpriseGradeValuesData = [
+	{
+		title: 'SSO and SAML support',
+	},
+	{
+		title: 'Query API keys',
+	},
+	{
+		title: 'Advanced security with SOC 2 Type I certification',
+	},
+	{
+		title: 'AWS Private Link',
+	},
+	{
+		title: 'VPC peering',
+	},
+	{
+		title: 'Custom integrations',
+	},
+];
+
+export const customerStoriesData = [
+	{
+		key: 'c-story-1',
+		avatar: 'https://signoz.io/img/users/subomi-oluwalana.webp',
+		personName: 'Subomi Oluwalana',
+		role: 'Founder & CEO at Convoy',
+		customerName: 'Convoy',
+		message:
+			"We use OTel with SigNoz to spot redundant database connect calls. For example, we found that our database driver wasn't using the connection pool even though the documentation claimed otherwise.",
```

Comment:
use locale strings

Reply:
let me try this out

Reply:
@YounixM can you guide me the best practices to do this to locale strings ? Since this is user content and not application text or UX copy does this need to be in locale.